src/policyengine_data/data_donwload_upload.py

```diff
@@ -0,0 +1,185 @@
+"""
```

Comment:
Would rather we took the opportunity to clean this up. We have two cloud locations (hf/gcp), and can default to Hugging Face. We should drop anything related to storing in github releases.

Can you write this module to just have `download()` and `upload()` main functions, which apply smaller functions `identify_location() -> str[HUGGING_FACE, GOOGLE_CLOUD_STORAGE]`, that apply `validate_hugging_face_url` etc.?

The checking for `hf://` etc is ugly (I know, I wrote it) and these functions are too large. Lets break them up